models/multiphase/d3q27_pf_velocity/Dynamics.R

```diff
@@ -62,8 +51,18 @@ if (Options$altContactAngle){
 if (Options$OutFlow){
 	for (d in rows(DensityAll)) {
 		AddField( name=d$name, dx=-d$dx-1, dy=-d$dy, dz=-d$dz )
+		AddField( name=d$name, dx=-d$dx+1, dy=-d$dy, dz=-d$dz )
+		## AddField( name=d$name, dx=-d$dx, dy=-d$dy-1, dz=-d$dz )
```

Comment:
I have removed these to avoid confusion. A comment has been added to .c.Rt that if for some reason more outflow conditions that aren't E or W are required - then they need to be implemented by the user using existing code as template